packages/communicator-email-ses/dbos-config.yaml

```diff
@@ -0,0 +1,25 @@
+# To enable auto-completion and validation for this file in VSCode, install the RedHat YAML extension
+# https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml
+
+# yaml-language-server: $schema=https://raw.githubusercontent.com/dbos-inc/dbos-transact/main/dbos-config.schema.json
+
+database:
+  hostname: localhost
+  port: 5432
+  username: postgres
+  password: ${PGPASSWORD}
+  app_db_name: hello
+  connectionTimeoutMillis: 3000
+  app_db_client: knex
+  migrate:
+    - npx knex migrate:latest
+  rollback:
+    - npx knex migrate:rollback
+application:
+  aws_ses_configurations: aws_config
+  ses_from_address: ${SES_FROM_ADDRESS}
+  ses_to_address: ${SES_TO_ADDRESS}
```

Comment:
Is this the recipient email address? Must it be fixed per app deploy? Can I make an app that emails to different addresses (once per API call for example). I have a use case for that :)

Reply:
This config file is for the unit test only.  You're the second one to be confused by this, I guess it needs a new file name?

Reply:
I renamed the config file, which I think is the right thing to do.  However, it is leading to a complaint about something being deprecated.